crates/wasmcloud-provider/Cargo.toml

```diff
@@ -22,23 +22,27 @@ rustls-tls = ["kube/rustls-tls", "kubelet/rustls-tls", "krator/rustls-tls"]
 [dependencies]
 anyhow = "1.0"
 async-trait = "0.1"
-wascc-host = "0.13"
+# wasmcloud-host = "0.16"
```

Comment:
This and the next line will be swapped out once the bugfix release of wasmcloud-host is cut

Reply:
@thomastaylor312 did all of your testing work fine with the current `main` branch for wasmcloud-host? If so, we should be ready to bump and release

Reply:
Yep, this is good to go

Reply:
Yep, this is good to go